js/apps/admin-ui/src/user/UserAttributes.tsx

```diff
@@ -16,13 +19,13 @@ export const UserAttributes = ({ user, save }: UserAttributesProps) => {
   return (
     <PageSection variant={PageSectionVariants.light}>
       <AttributesForm
-        form={form}
+        form={form as UseFormReturn<AttributeForm>}
```

Comment:
this isn't very typesafe

Reply:
True, if you have a suggestion on how to handle this better let me know.